src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -218,21 +234,106 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
         label2: sprintf(lstrings.fiat_plugin_amount_currencycode, currencyCode),
         swapInputLocations: requireCrypto,
         async onChangeText() {},
+        async onMax(sourceFieldNum, stateManager) {
+          // Helper function to get a quote or error for the value specified
+          const getMaxQuoteOrError = async (
```

Comment:
```suggestion
          /** Helper function to get a quote or error for the value specified */
```

Using this comment format instead will show the comment on hover